core/phases.py

```diff
@@ -117,5 +121,13 @@ def end(self) -> PhaseState:
 class FeedingPhase(Phase):
     def __init__(self, game):
         super().__init__(game)
-        dispatcher.send(signal=Signal.START_FEEDING_PHASE, sender=self)
 
+    def start(self):
```

Comment:
What calls this method?

Reply:
I am also wondering if we should not make certain signals class static members of the phases? E.g. if we have signals that relate to the start/end of phases, maybe we should make that explicit?

Reply:
Also, will we ever want to do something in between initialising a phase and 'Start'ing it? If not, then maybe we should just use the constructor for that purpose?